### PR TITLE
Changes to get rubocop & rspec to pass

### PR DESCRIPTION
**Pull request by SCI4THIS**
```
Fix rspec tests.

* .gitignore
** Looks like application.yml was intended to be included as per
** commit 2959c19d17d3f637508413aa85c2d8818fdc7b09
** it was in .gitignore twice so that commit only deleted one of the
** references.  This deletes the second reference.

* config/application.yml
** This adds the application.yml that I think was intended to be added
** in the above commit.  This was application.example.yml in the commit.
** The MAILER_SENDER & LOAD_MAILER_RECIPIENTS values are needed to get
** rspec to pass (They can also be set via environment, but this seems like
** a better way).

Fix rubocop issues.

* .rubocop.yml
** https://github.com/bbatsov/rubocop/blob/master/CHANGELOG.md
** These changes broke the yml: #1020, #1397, #1600
** Updated yml to pass with new changes.
*** May need further configuration.

* spec/features/loads/user_views_a_list_of_loads_spec.rb
** Roll up logic from unrolled loop to accomodate rubocop ABC 15 limit.
```

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -645,7 +644,7 @@ Style/BlockComments:
 Style/BlockEndNewline:
   Description: Put end statement of multiline block on its own line.
   Enabled: true
-Style/Blocks:
+Style/BlockDelimiters:
```
Comment (josephbridgwaterrowe) [inline .rubocop.yml]: @okibar, I'm curious why you changed these, was it because of updates to rubocop?

Thanks.

File: .rubocop.yml [hunk 2]
```diff
@@ -923,7 +922,7 @@ Style/UnneededPercentQ:
   Description: Checks for %q/%Q when single quotes or double quotes would do.
   StyleGuide: https://github.com/bbatsov/ruby-style-guide#percent-q
   Enabled: true
-Style/UnneededPercentX:
+Style/CommandLiteral:
```
Comment (josephbridgwaterrowe) [inline .rubocop.yml]: @okibar, same question here.

File: .rubocop.yml (hunk 1 above)
Comment (SCI4THIS) [inline .rubocop.yml]: This one fixes this warning
Warning: unrecognized cop Style/Blocks found in /Users/a/Desktop/westmill/simple_loads/.rubocop.yml
which was introduced in rubocop 0.30.0
The Gemfile.lock specifies 0.34.2

File: .rubocop.yml (hunk 2 above)
Comment (SCI4THIS) [inline .rubocop.yml]: UnneededPercentX & MaxSlashes were changed in rubocop 0.30.0 as well

Comment (SCI4THIS) [inline .rubocop.yml]: Maybe the real fix is to change the Gemfile.lock to 0.29.xxx ?

**Pull request closed without merging**